### ISSUE TITLE
Europe Mode needs a way to revert

### ISSUE THREAD

**Issue by trlkly** [rcmaehl/MSEdgeRedirect#382]
### Description of the new feature / enhancement

The new Europe Mode option is a great idea. But any action taken by MSEdgeRedirect should be able to be undone, in a logical manner.

### Scenario when this would be used?

Any of the same reasons someone might uninstall MSEdgeRedirect. They could have weird problems with it, or need to return to stock. They may actually decide to use Edge. Microsoft might detect what is being done. 

The exact reason doesn't matter: one always expects that they can revert any settings change software makes. It's the accepted practice. 

### Supporting information

The most logical way to do this would to backup their old settings and allow them to undo them by running the software again. However, it might also be good to allow people to set non-European countries in case they tried it not knowing they couldn't revert it--like I did.

**Comment by a562314** [comment]
Yes, I'm also bothered by the fact that I can't undo the changes

**Comment by Reproved751** [comment]
https://github.com/rcmaehl/MSEdgeRedirect/issues/392#issuecomment-1953658174 They Just fixed it for me You can just translate the text @a562314

**Comment by micwoj92** [comment]
Closing as duplicate of #392 since it contains more activity.
